packages/cheqd/src/dids/CheqdDidRegistrar.ts

```diff
@@ -196,6 +222,12 @@ export class CheqdDidRegistrar implements DidRegistrar {
             keyType: KeyType.Ed25519,
             privateKey: privateKey,
           })
+          const contextUrl = this.contextMapping[verificationMethod.type as keyof typeof this.contextMapping]
```

Comment:
This should be moved below maybe after the update operation. You can see there are some additions to the verificationMethod in the below step.

Reply:
Agreed, this is done.